src/vs/workbench/services/textfile/common/textFileEditorModelManager.ts

```diff
@@ -24,6 +24,8 @@ export class TextFileEditorModelManager implements ITextFileEditorModelManager {
 
 	private toUnbind: IDisposable[];
 
+	private _onModelDisposed: Emitter<URI>;
```

Comment:
@Tyriar can we have test coverage for `onModelDisposed` and `onModelContentChanged`